src/test/java/org/apache/sysds/test/functions/builtin/BuiltinMiceTest.java

```diff
@@ -50,33 +50,47 @@ public void testMiceMixCP() {
 		runMiceNominalTest(mask, 1, LopProperties.ExecType.CP);
 	}
 
+//	@Test
+//	public void testMiceMixSpark() {
+//		double[][] mask = {{ 0.0, 0.0, 1.0, 1.0, 0.0}};
+//		runMiceNominalTest(mask, 1, LopProperties.ExecType.SPARK);
+//	}
+
 	@Test
 	public void testMiceNumberCP() {
 		double[][] mask = {{ 0.0, 0.0, 0.0, 0.0, 0.0}};
 		runMiceNominalTest(mask, 2, LopProperties.ExecType.CP);
 	}
 
+//	@Test
+//	public void testMiceNumberSpark() {
+//		double[][] mask = {{ 0.0, 0.0, 0.0, 0.0, 0.0}};
+//		runMiceNominalTest(mask, 2, LopProperties.ExecType.SPARK);
+//	}
+
 	@Test
 	public void testMiceCategoricalCP() {
 		double[][] mask = {{ 1.0, 1.0, 1.0, 1.0, 1.0}};
 		runMiceNominalTest(mask, 3, LopProperties.ExecType.CP);
 	}
-	//	@Test
-	//	public void testMiceSpark() {
-	//		runMiceNominalTest( LopProperties.ExecType.SPARK);
-	//	}
+
+//	@Test
+//	public void testMiceCategoricalSpark() {
+//		double[][] mask = {{ 1.0, 1.0, 1.0, 1.0, 1.0}};
+//		runMiceNominalTest(mask, 3, LopProperties.ExecType.SPARK);
+//	}
```

Comment:
Could you mark these tests with @Ignore so that our test framework skips them, but reports that they are skipped. This will notify us that there is a problem we need to fix.